GeneratorAnalysis/Scripts/GeneratorInteBreakDown.cpp

```diff
@@ -193,7 +205,19 @@ void GeneratorInteBreakDown() {
     PlotNames.push_back("TrueDeltaAlphaTPlot");
     XAxisLabel.push_back("#delta #alpha_{T} [deg]");
     YAxisLabel.push_back("#frac{d#sigma}{d#delta #alpha_{T}} #left[10^{-38} #frac{cm^{2}}{Ar}#right]");
-
+	
+    PlotNames.push_back("TrueInvariantMassPlot");
+    XAxisLabel.push_back("W [GeV]");
```

Comment:
You do not have delta here but have it above, should it be with or without delta throughout?